plugins/chat/lib/chat_channel_hashtag_data_source.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Chat::ChatChannelHashtagDataSource
+  def self.icon
+    "comment"
+  end
+
+  def self.channel_to_hashtag_item(guardian, channel)
+    HashtagAutocompleteService::HashtagItem.new.tap do |item|
+      item.text = channel.title(guardian.user)
+      item.slug = channel.slug
+      item.icon = icon
+      item.url = channel.url
```

Comment:
Could you check what kind of URL we are storing here? wonder if relative or absolute, we should go for relative, I think this is what we do for tags/categories

Reply:
Yes will double check, relative is probably better, I will make it explicit that it is `relative_url` too